receiver/podmanreceiver/metadata.yaml

```diff
@@ -8,3 +8,119 @@ status:
   codeowners:
     active: [rogercoll]
 
+resource_attributes:
+  container.runtime:
+    description: "The runtime of the container. For this receiver, it will always be 'podman'."
+    type: string
+    enabled: true
+  container.id:
+    description: "The ID of the container."
+    type: string
+    enabled: true
+  container.image.name:
+    description: "The name of the image in use by the container."
+    type: string
+    enabled: true
+  container.name:
+    description: "The name of the container."
+    type: string
+    enabled: true
+
+attributes:
+  core:
+    description: "The CPU core number when utilising per-CPU metrics."
+    type: string
+
+metrics:
+  # CPU
+  container.cpu.usage.system:
+    enabled: true
+    description: "System CPU usage."
+    unit: ns
```

Comment:
https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemcputime
I'm not sure if cputime unit should be `ns` or `s`, @open-telemetry/collector-approvers should this metric use `s` as unit?

Reply:
I think that cputime metric's unit is in seconds, as this is how many OSes report it (/proc/stats). But containers are handled by cgroup controllers, which allow for [better precision](https://www.kernel.org/doc/Documentation/cgroup-v2.txt). I would say not to lose precision, instead add container metrics into the semantic convention.

Reply:
Your comments also make sense, I will pull this discussion to slacks, to get more input. @rogercoll

Reply:
The docker stats receiver collects `container.cpu.usage.system` as well: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/metadata.yaml#L63-L67, and the unit is `ns`. IMO we should also use `ns` here to stay consistent between the two.

Reply:
Sounds good! Does it make sense to start working on the container's semantic convention PR? (feel free to add me into the slack thread _@rogercoll_)

Reply:
No one responses for that for a day. And as @mackjmr  said docker stats receiver also take `ns` as unit, so I think this is ok to approve.